kickstart/kickstart.json

```diff
@@ -16,6 +16,9 @@
       "url": "/api/application/85a03867-dccf-4882-adde-1a79aeec50df",
       "body": {
         "application": {
+  	  "oauthConfiguration": {
+            "clientSecret" : "6HvlAe28V4PL9OEKJ0VlZ6ZsEAluEsZHVFYKuoLAr1U"
```

Comment:
Ideally this would actually use the value from the `config.js` file instead of just copying it. Kickstart can read environment variables, so you could use the value from `config.js` to set an environment variable and then use it here so that they are in sync.

Reply:
Hmmm. The "right way" to set environment variables in React is to use a .env file. But if I do that, should I move the whole config.js to that system?